internal/nostr/nostr.go

```diff
@@ -139,7 +148,11 @@ func NewService(ctx context.Context) (*Service, error) {
 		// Create a copy of the loop variable to
 		// avoid passing address of the same variable
     subscription := sub
-		go svc.startSubscription(svc.Ctx, &subscription, nil, svc.handleSubscribedEvent)
+		handleEvent := svc.handleSubscribedEvent
+		if sub.PushToken != "" {
+			handleEvent = svc.handleSubscribedExpoNotification
```

Comment:
Can the naming be improved here? we are still handling a subscribed event, right - we're just notifying in a different way?

Reply:
Changed to be simple - `handlePushNotification`, wdyt?

Reply:
I think it's still inconsistent with `handleSubscribedEvent`

Reply:
A bit long but changed to `handleSubscribedEventForPushNotification`